.github/.jira_sync_config.yaml

```diff
@@ -17,8 +22,11 @@ settings:
   components:
     - smartctl-exporter
 
+  # (Optional) (Default: false) Add a new comment in GitHub with a link to Jira created issue
+  add_gh_comment: true
+
   # (Optional) (Default: None) Parent Epic key to link the issue to
-  epic_key: SOLENG-190
+  epic_key: SOLENG-46
```

Comment:
Not sure if we should change this number

Reply:
@chanchiwai-ray was this an intentional change?

Reply:
updated